src/app.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
please start with camel case for files with components

Reply:
Thanks for your note.